dashboard/app/controllers/home_controller.rb

```diff
@@ -64,4 +84,29 @@ def terms_and_privacy
   def teacher_announcements
     render template: 'api/teacher_announcement', layout: false
   end
+
+  def recent_courses
+    current_user.in_progress_and_completed_scripts.map do |script|
+      script_id = script[:script_id]
+      script_name = Script.find_by_id(script_id)[:name]
+      {
+        id: script_id,
+        script_name: script_name,
+        courseName: data_t_suffix('script.name', script_name, 'title'),
+        description: data_t_suffix('script.name', script_name, 'description_short'),
+        link: script_url(script_id),
+        image: "",
+        assignedSections: []
+      }
+    end
+  end
+
+  def courses
+    if current_user
+      @recent_courses = recent_courses
+
+      # Is the language english and the user a teacher?  (No, not are they actually an English teacher.)
+      @english_teacher = current_user.teacher? && request.language == 'en'
```

Comment:
will we ever need to know if non-english users are students or teachers? i wonder if it makes more sense to just have this as two separate fields.